DEVELOPING.md

```diff
@@ -3,24 +3,27 @@ We appreciate any contributions to the Score-P Python Bindings.
 However, there are a few policies we agreed on and which are relevant for contributions.
 These are detailed below. 
 
-## Fromatting / Codestyle
+## Formatting / Codestyle
 
 Readable and consistent code makes it easy to understand your changes.
-Therefore the GitHub CI system has checks using `clang-format-9` and `flake8` in place.
```

Comment:
I think it is not required which CI system is used. Allows us more freedom to change it